src/index.html

```diff
@@ -1,5 +1,8 @@
 <!doctype html>
-<html lang="en">
+<html
```

Comment:
The `<html>` tag is incorrectly self-closed. HTML tags such as `<html>` should not be self-closing. Remove the `/` at the end of the tag to fix this issue.